src/test/java/it/gov/pagopa/pu/sil/mapper/PaaSILInviaDovutiMapperTest.java

```diff
@@ -0,0 +1,178 @@
+    assertEquals(SilFaults.PAA_IUV_NON_VALIDO, result.getMiddle());
+    assertEquals("L'inserimento dello IUV è deprecato", result.getRight());
+  }
+
+  @Test
+  void mapRequestToDebtPositionsOrFault_InvalidDebtor_ReturnsError() {
+    PaaSILInviaDovuti request = new PaaSILInviaDovuti();
+    Dovuti dovuti = podamFactory.manufacturePojo(Dovuti.class);
+    dovuti.getDatiVersamento().setIdentificativoUnivocoVersamento(null);
+    dovuti.setSoggettoPagatore(new CtSoggettoPagatore());
+    when(jaxbTransformServiceMock.unmarshalling(any(), eq(Dovuti.class))).thenReturn(dovuti);
+    when(organizationServiceMock.getOrganizationById(anyLong(), anyString())).thenReturn(Optional.of(org));
+    when(personMapperMock.getAndValidateDebtor(any())).thenReturn(Triple.of(null, SilFaults.PAA_ANAGRAFICA_NON_VALIDA, "Error"));
+
+    Triple<List<DebtPositionDTO>, SilFaults, String> result = mapper.mapRequestToDebtPositionsOrFault(request, userInfo, ORG_IPA_CODE, ACCESS_TOKEN);
+
+    assertNotNull(result);
+    assertNull(result.getLeft());
+    assertEquals(SilFaults.PAA_ANAGRAFICA_NON_VALIDA, result.getMiddle());
+    assertEquals("Error", result.getRight());
+  }
+
+  @Test
+  void mapRequestToDebtPositionsOrFault_ValidFlow_ReturnsDebtPositions() {
+    PaaSILInviaDovuti request = new PaaSILInviaDovuti();
+    Dovuti dovuti = podamFactory.manufacturePojo(Dovuti.class);
+    dovuti.setSoggettoPagatore(new CtSoggettoPagatore());
+    CtDatiSingoloVersamentoDovuti versamento = new CtDatiSingoloVersamentoDovuti();
+    versamento.setCausaleVersamento("causale");
+    versamento.setImportoSingoloVersamento(BigDecimal.TWO);
+    versamento.setDatiSpecificiRiscossione("9/DATI_SPECIFICI");
+    versamento.setIdentificativoTipoDovuto("COD_TIPO_DOVUTO");
+    versamento.setIdentificativoUnivocoDovuto("IUD");
+    dovuti.getDatiVersamento().getDatiSingoloVersamentos().clear();
+    dovuti.getDatiVersamento().getDatiSingoloVersamentos().add(versamento);
+    dovuti.getDatiVersamento().setIdentificativoUnivocoVersamento(null);
+    PersonDTO debtor = podamFactory.manufacturePojo(PersonDTO.class);
+    when(jaxbTransformServiceMock.unmarshalling(any(), eq(Dovuti.class))).thenReturn(dovuti);
+    when(organizationServiceMock.getOrganizationById(anyLong(), anyString())).thenReturn(Optional.of(org));
+    when(personMapperMock.getAndValidateDebtor(any())).thenReturn(Triple.of(debtor, null, null));
+    when(debtPositionServiceMock.getDebtPositionTypeOrgByOrgIdAndType(
+      org.getOrganizationId(), versamento.getIdentificativoTipoDovuto(), ACCESS_TOKEN)).thenReturn(podamFactory.manufacturePojo(DebtPositionTypeOrg.class));
+    Triple<List<DebtPositionDTO>, SilFaults, String> result = mapper.mapRequestToDebtPositionsOrFault(request, userInfo, ORG_IPA_CODE, ACCESS_TOKEN);
+
+    assertNotNull(result);
+    assertNotNull(result.getLeft());
+    assertNull(result.getMiddle());
+    assertNull(result.getRight());
+    assertEquals(1, result.getLeft().size());
+  }
```

Comment:
let's add checkNotNull fields on each hierarchy entity of the DebtPosition in order to explicity see what's expected to be null